todo/src/components/Form.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import {addTask} from '../actions';
+import './todo.css'
+
+class Form extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            todo: ''  
+    }}
```

Comment:
You could add the appropriate line spaces and tabbing for your closing curly braces `}` to improve readability here.